client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -894,7 +894,12 @@
                 $scope.close = function() {
                     _closing = true;
                     authoring.close($scope.item, $scope.origItem, $scope.dirty).then(function() {
-                        $location.url($scope.referrerUrl);
+                        if (sessionStorage.getItem('previewUrl')) {
+                            $location.url(sessionStorage.getItem('previewUrl'));
+                            sessionStorage.removeItem('previewUrl');
```

Comment:
Yeah, it removes value from sessionStorage... Maybe is good for transparency that I move it before $location change. I will make this change when I finish writting e2e/unt tests.